sklearn/metrics/tests/test_common.py

```diff
@@ -1743,20 +1744,22 @@ def test_metrics_pos_label_error_str(metric, y_pred_threshold, dtype_y_str):
 
 
 def check_array_api_metric(
-    metric, array_namespace, device, dtype_name, y_true_np, y_pred_np, sample_weight
+    metric, array_namespace, device, dtype_name, x_np, y_np, **metric_kwargs
```

Comment:
I'm not sure if `x` and `y` are good names for these 2 arrays (because in sklearn most of the time x is for features, and y for labels)

I just wanted to make this more generic so that can be used for pairwise metrics too, where the 2 arrays are not `y_pred` and `y_true`.

Any better idea? maybe `a` and `b`?

Reply:
Yes, I thought the same. But then I realized we already make this naming "mistake" in the prototype of pairwise metrics.

> Any better idea? maybe a and b?

+1.